firestore-bigquery-export/functions/package.json

```diff
@@ -18,7 +18,7 @@
     "@types/chai": "^4.1.6",
     "chai": "^4.2.0",
     "firebase-admin": "^8.1.0",
```

Comment:
I think the Admin SDK needs updating also here and is what's causing the failure